src/components/Scroll.js

```diff
@@ -8,6 +8,23 @@ import "../css/Scroll.scss";
  * Taken from https://reacttraining.com/react-router/web/guides/scroll-restoration
  */
 class Scroll extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { width: 0 };
+    this.updateWindowDimensions = this.updateWindowDimensions.bind(this);
```

Comment:
Instead of binding in the constructor, you can change `updateWindowDimensions` to an arrow function. This allows you to delete the entire constructor, so you can place the state directly in the class:

```js
class Scroll extends Component {
  ...
  state = { width: 0 }
  ...
}
```